packages/vm/src/parsed_wasm.rs

```diff
@@ -30,13 +30,14 @@ pub struct ParsedWasm<'a> {
 impl<'a> ParsedWasm<'a> {
     pub fn parse(wasm: &'a [u8]) -> VmResult<Self> {
         let mut validator = Validator::new_with_features(WasmFeatures {
-            deterministic_only: true,
             component_model: false,
             simd: false,
             relaxed_simd: false,
             threads: false,
             multi_memory: false,
             memory64: false,
+            gc: false,
+            memory_control: false,
             ..Default::default()
```

Comment:
Could we roll out the fields here to ensure we see all? There is e.g. `bulk_memory` which we do not support in CosmWasm.

https://docs.rs/wasmparser/latest/src/wasmparser/validator.rs.html#369-396